fmriprep/workflows/bold/confounds.py

```diff
@@ -412,10 +412,22 @@ def init_bold_confs_wf(
         name='crowncc_metadata_fmt',
     )
 
+    # Combine all confounds metadata
     mrg_conf_metadata = pe.Node(
-        niu.Merge(3), name='merge_confound_metadata', run_without_submitting=True
+        niu.Merge(4), name='merge_confound_metadata', run_without_submitting=True
     )
+    # Tissue mean time series
     mrg_conf_metadata.inputs.in3 = {label: {'Method': 'Mean'} for label in signals_class_labels}
+    # Movement parameters
+    mrg_conf_metadata.inputs.in4 = {
+        'trans_x': {'Description': 'Translation Along X Axis', 'Units': 'mm'},
+        'trans_y': {'Description': 'Translation Along Y Axis', 'Units': 'mm'},
+        'trans_z': {'Description': 'Translation Along Z Axis', 'Units': 'mm'},
+        'rot_x': {'Description': 'Rotation Along X Axis', 'Units': 'rad'},
+        'rot_y': {'Description': 'Rotation Along Y Axis', 'Units': 'rad'},
+        'rot_z': {'Description': 'Rotation Along Z Axis', 'Units': 'rad'},
```

Comment:
Should be rotation "about". And could we indicate the x axis is left-right, and so on?

Reply:
Absolutely can do. Just to make sure, the motion parameters don't depend on the phase encoding direction? Meaning they don't directly map onto i, j, and k at this point?

Reply:
Hmmm, would be good to check.